app/views/questions/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= form_for @question do |f| %>
+  <%= f.text_field :title, placeholder: "Title" %>
+  <%= f.text_area :body, placeholder: "Body" %>
+  <%= f.label :category %>
+  <%= f.collection_select(:category_id, @categories, :id, :name, class: "browser-default") %>
```

Comment:
@TheKaterTot Per our conversation, this looks good to me. It's consistent with what I'm seeing in both the Rails docs and API dock. Let me know if you got this figured out after we spoke.